### PR TITLE
Add writeup on Ember performance tweaks

### PR DESCRIPTION
**Pull request by amyrlam**
@abhilashlr wanted to check that you prefer to be called by `Abhilash LR` and your pronoun is `he/him`? let me know!

great post! 💫

### CODE REVIEW

File: source/2020-05-15-the-ember-times-issue-148.md [hunk 1]
```diff
@@ -55,14 +57,19 @@ Check out the full video explaining the `{{on}}` modifier on [EmberMap](https://
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Ember performance tweaks 🏎](https://abhilashlr.in/ember-performance-tweaks-part-1)
+  
+[Abhilash LR (@abhilashlr)](https://github.com/abhilashlr) blogs about **optimising build timelines & bundle size** in your Ember apps, the first in a series on getting started with performance optimizations! Check out his tips and tricks in the writeup [here](https://abhilashlr.in/ember-performance-tweaks-part-1). Highlights include:
```
Comment (ijlee2) [inline source/2020-05-15-the-ember-times-issue-148.md]: I noticed that you used `z`  for other words with s/z variance so I think it'd be good to use the American spelling consistently.

```suggestion
[Abhilash LR (@abhilashlr)](https://github.com/abhilashlr) blogs about **optimizing build timelines & bundle size** in your Ember apps, the first in a series on getting started with performance optimizations! Check out his tips and tricks in the writeup [here](https://abhilashlr.in/ember-performance-tweaks-part-1). Highlights include:
```

**Review by ijlee2 — APPROVED** [review]
Looks good to me! I think we (whoever takes it) can cover the 2nd part of Abhilash's performance series after your quote in the end. For example,

```
> The smaller the page assets, the faster it is for the user to view them. The faster it is for the user to view the page with text and UI, the greater trust they have on our apps/websites.

But wait, there's more! ...
```

**Comment by jayjayjpg** [comment]
> Looks good to me! I think we (whoever takes it) can cover the 2nd part of Abhilash's performance series after your quote in the end. For example,

@ijlee2 @amyrlam Good idea, I can do an extension with a writeup on the second blog post for example!

**Comment by ijlee2** [comment]
@jayjayjpg Sounds good. Feel free to cover the 2nd part, thanks!

File: source/2020-05-15-the-ember-times-issue-148.md (hunk 1 above)
Comment (amyrlam) [reply source/2020-05-15-the-ember-times-issue-148.md]: @ijlee2 aha, good catch!

**Comment by amyrlam** [comment]
thank you both! @jayjayjpg @ijlee2

**Comment by abhilashlr** [comment]
Hi @amyrlam, What you have shared is correct 👍 
And thanks for writing on it 😄

**Comment by abhilashlr** [comment]
Thank you both @jayjayjpg and @ijlee2 :)

**Pull request merged**